lib/travis/build/script/nix.rb

```diff
@@ -43,19 +43,13 @@ def configure
         def setup
           super
 
-          version = config[:nix]
+          version = config[:nix] == "latest" ? config[:nix] : "nix-#{config[:nix]}"
 
           sh.fold 'nix.install' do
-            sh.cmd "wget --retry-connrefused --waitretry=1 -O /tmp/nix-install https://nixos.org/releases/nix/nix-#{version}/install"
+            sh.cmd "wget --retry-connrefused --waitretry=1 -O /tmp/nix-install https://nixos.org/releases/nix/#{version}/install"
             sh.cmd "yes | sh /tmp/nix-install"
 
-            if config[:os] == 'linux'
-              # single-user install (linux)
-              sh.cmd 'source ${TRAVIS_HOME}/.nix-profile/etc/profile.d/nix.sh'
-            else
-              # multi-user install (macos)
-              sh.cmd 'source /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh'
-            end
+            sh.cmd 'source /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh'
```

Comment:
@matthewbauer this path does not exist on my NixOS 19.09 installation, do you know if the above will work?

Reply:
Yeah, it won't exist on NixOS because it has its own version of the nix-daemon.sh script. You can install it via `sudo nix-env -iA nixos.nix`.

But it should exist on anything that has run `install-multi-user.sh`.